ksp_plugin/part.cpp

```diff
@@ -228,7 +248,7 @@ std::string Part::ShortDebugString() const {
 std::ostream& operator<<(std::ostream& out, Part const& part) {
   return out << "{"
              << part.part_id() << ", "
-             << part.mass() << "}";
+             << part.inertia_tensor().mass() << "}";
```

Comment:
Log the tensor too.

Reply:
As discussed, let's not do that, this is not even used.